MonkeyLoader.Resonite.Integration/Configuration/SettingsDataFeedInjector.cs

```diff
@@ -232,68 +281,61 @@ await foreach (var feedItem in EnumerateMonkeysAsync(path, mod, EarlyMonkeys))
 
         private static async IAsyncEnumerable<DataFeedItem> EnumerateModsAsync(IReadOnlyList<string> path)
         {
-            var monkeyLoaderCategory = new DataFeedCategory();
-            monkeyLoaderCategory.InitBase("MonkeyLoader", path, null, Mod.GetLocaleString("OpenMonkeyLoader.Name"), Mod.GetLocaleString("OpenMonkeyLoader.Description"));
-            yield return monkeyLoaderCategory;
+            var modsGroup = new DataFeedGroup();
+            modsGroup.InitBase("Mods.Group", path, null, Mod.GetLocaleString("Mods"));
```

Comment:
If possible maybe reduce the number of buttons per row?

Reply:
Doesn't appear to be possible without fuckery (maybe looking for the grid component and changing that?)